### ISSUE TITLE
Feature Request: Import Non-Voxel Models as References

### ISSUE THREAD

**Issue by SVLubin** [guillaumechereau/goxel#407]
First, thank you for creating such an amazing voxel modelling tool! It’s been incredibly helpful, and I appreciate all the effort you’ve put into it.

I would like to suggest a feature to enhance the tool: the ability to import non-voxel models (e.g., OBJ, STL) as references. This would allow users to use these models as guides for designing voxel-based versions by building around them.

Please let me know if you’d like more details or examples of how this feature could be beneficial. I’d be happy to help!

**Comment by guillaumechereau** [comment]
Yes I think something like that could be useful indeed.  I’ll need to see
how difficult it would be to implement.
